### PR TITLE
timeline updates

### PR DESCRIPTION
**Pull request by swagatmitra-b**

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -106,6 +106,17 @@ function Navbar() {
               Team
             </Link>
           </li>
+          <li className="nav-button blog">
+            <a
+              href="https://gdscnits.hashnode.dev"
```
Comment (byt3h3ad) [inline src/components/Navbar/Navbar.jsx:111]: add a target `_blank`

**Comment by byt3h3ad** [comment]
image
fix the responsive issue and we'll be good to go. nice work!

**Comment by swagatmitra-b** [comment]
I've addressed the issues! Please review them :)

**Comment by byt3h3ad** [comment]
instead of the direct link to the blog, can you see if `/blog` in the navbar works? this will be useful in the future, if the link needs to be updated. the change needs to be done in one place only.

**Comment by swagatmitra-b** [comment]
I've replaced the <a> tag with a <Link> tag from react router which points to `/blog`. Now this obviously doesn't work locally since we don't have a `/blog` route but following the changes in #113, due to the _redirects file in the public folder which redirected `gdscnits.tech/blog` to the actual blog page, I think this should work in production even though I'm not entirely sure about it.

**Comment by byt3h3ad** [comment]
yeah works. gg!

**Pull request merged**